systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1450,6 +1457,157 @@ void testClientsCACertRenew() {
                 initialKafkaUserCertEndTime.compareTo(changedKafkaUserCertEndTime) < 0);
     }
 
+    static final String STRIMZI_TEST_CLUSTER_CA = "C=CZ, L=Prague, O=StrimziTest, CN=SecuritySTClusterCA";
+    static final String STRIMZI_TEST_CLIENTS_CA = "C=CZ, L=Prague, O=StrimziTest, CN=SecuritySTClientsCA";
```

Comment:
Move this to top of the class, it's not very clear when you have this stuff in the middle of the class